backend/fms_core/models/_validators.py

```diff
@@ -14,3 +15,5 @@
 container_barcode_validator = RegexValidator(re.compile("^[\S]{1,200}$"))
 email_validator = EmailValidator()
 sequence_validator = RegexValidator(re.compile("^[ATCGU]{0,500}$"))
+metadata_name_validator = RegexValidator(regex=re.compile(r"^(?!.*__)[a-zA-Z0-9.\-_]{1,200}$"),
+                                         message="Double underscore i.e '__' are not allowed when naming metadata fields.")
```

Comment:
This message is displayed even when the invalid characters are not the '__', you should point out the valid characters too to make sure the person knows what was wrong with it's name (make it a general message).